controllers/levels.js

```diff
@@ -0,0 +1,66 @@
+const LevelModel = require("../models/levels");
+
+/**
+ * Creates a level
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const addLevel = async (req, res) => {
+  try {
+    const { id, levelData } = await LevelModel.addLevel(req.body);
```

Comment:
can we get the data of the person who created the tag using authenticate middleware?

Reply:
done